centreon/unattended.sh

```diff
@@ -515,16 +538,10 @@ function secure_mariadb_setup() {
 function install_centreon_repo() {
 
 	log "INFO" "Centreon official repositories installation..."
-	$PKG_MGR -q clean all
 
-	rpm -q centreon-release-$CENTREON_MAJOR_VERSION >/dev/null 2>&1
+	$PKG_MGR config-manager --add-repo $RELEASE_RPM_URL
```

Comment:
This should be named properly with RELEASE_REPO_FILE for instance to better show the purpose.